pkg/corerp/frontend/controller/environments/createorupdateenvironment.go

```diff
@@ -177,7 +177,7 @@
 			resourceGroupName,
 			vnetName,
 			armnetwork.VirtualNetwork{
-				Location: to.Ptr(ResourceLocation),
+				Location: resourceLocation,
```

Comment:
The consistent replacement of to.Ptr(ResourceLocation) with resourceLocation across all Azure resource creation calls (lines 180, 213, 286) is correct. Since GetResourceGroupLocation() returns a pointer to string, this maintains type consistency.